src/js/main.js

```diff
@@ -42,6 +42,24 @@ export function init(el, context, config) {
             hideAboutBtn.classList.remove('docs--show-about');
         };
 
+
+        // Show the trailer on click #docs__playTrailer
+        const $showTrailerAll = builder.querySelectorAll('.docs__shows-trailer');
+        [].forEach.call($showTrailerAll, function($showTrailer) {
```

Comment:
are we going to have more than one trailer on a page? if not, we could simplify this by removing the `forEach` in favour of using a single element